tgui/packages/tgui/interfaces/PaperSheet.js

```diff
@@ -358,24 +356,22 @@ class PaperSheetStamper extends Component {
       rotate: this.state.rotate,
     };
     return (
-      <Box
-        onClick={this.handleMouseClick.bind(this)}
-        onMouseMove={this.handleMouseMove.bind(this)}
-        onwheel={this.handleWheel.bind(this)} {...rest}>
+      <Fragment>
         <PaperSheetView
           readOnly
           value={value}
           stamps={stamp_list} />
         <Stamp
+          active_stamp
```

Comment:
Not a fan of `snake_case` prop names, prefer `camelCase`.

Reply:
You also have this single prop on one line, with the following (untouched) line having two props on it. Put all three on separate lines.